src/App.js

```diff
@@ -1,24 +1,14 @@
-import logo from './logo.svg';
+import Pokemon from './components/Pokemon/Pokemon';
+import PokemonDetail from './components/PokemonDetailPage/PokemonDetailPage';
```

Comment:
When you import a component, you want to be consistent in giving the name of import. For example: PokemonDetailPage should be fine and you also keep the name of the exported component.